test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -388,7 +388,6 @@ public void TagHelpers_Directive_GenerateDesignTimeMappings()
 
         [Theory]
         [InlineData("TagHelpersInSection")]
```

Comment:
Modify this test to just validate the section now, no need for a theory. Update test name accordingly.